autocorpus/autocorpus.py

```diff
@@ -48,36 +48,36 @@ def __soupify_infile(self, fpath):
             return soup
 
     def __get_keywords(self, soup, config):
-        if "keywords" in config:
-            responses = handle_not_tables(config["keywords"], soup)
-            responses = " ".join([x["node"].get_text() for x in responses])
-            if not responses == "":
-                keyword_section = {
-                    "section_heading": "keywords",
-                    "subsection_heading": "",
-                    "body": responses,
-                    "section_type": [
-                        {"iao_name": "keywords section", "iao_id": "IAO:0000630"}
-                    ],
-                }
-                return [keyword_section]
-            return False
+        if "keywords" not in config:
+            return None
+
+        responses = handle_not_tables(config["keywords"], soup)
+        if not responses:
+            return None
```

Comment:
This should probably return and empty dictionary instead of `None`

Reply:
Good point